test/e2e-test-application/e2e/tests/1-angular/luigi-client-link-manager-features.spec.js

```diff
@@ -259,4 +259,41 @@ describe('Luigi client linkManager', () => {
       cy.wrap($iframeBody).should('contain', $inputTypeModal);
     });
   });
+
+  describe.only('Drawer', () => {
+    let $iframeBody;
+    beforeEach(() => {
+      cy.visitLoggedIn('/projects/pr2');
+      cy.getIframeBody().then(result => {
+        $iframeBody = result;
+      });
+    });
+    it('Open and close drawer component with default settings', () => {
+      cy.goToLinkManagerMethods($iframeBody);
+      cy.expectPathToBe('/projects/pr2');
+      cy.get('.drawer').should('not.exist');
+      cy.wrap($iframeBody)
+        .contains('Open drawer with default settings')
+        .click();
+      cy.get('.drawer').should('exist');
+      cy.expectPathToBe('/projects/pr2');
```

Comment:
I'm not sure if cypress allows it, but would that be possible to move it to beforeEach block so it will be written once and reused in every test?

Reply:
👍  I moved the first three lines to the beforeEach block.. more is not possible.